Mathlib/Geometry/Convex/Cone/Pointed.lean

```diff
@@ -181,4 +181,16 @@ theorem toConvexCone_positive : ↑(positive R E) = ConvexCone.positive R E :=
   rfl
 
 end PositiveCone
+
+section OrderedAddCommGroup
+variable [Semiring R] [PartialOrder R] [IsOrderedRing R] [AddCommGroup E] [PartialOrder E]
+  [Module R E]
+
+/-- Constructs an ordered module given an ordered group, a cone, and a proof that
+the order relation is the one defined by the cone. -/
+lemma to_posSMulMono (C : PointedCone R E) (h : ∀ x y : E, x ≤ y ↔ y - x ∈ C) :
+    PosSMulMono R E where
```

Comment:
Can't we get an actual `IsOrderedModule` out of this...?

Reply:
Not without strengthening `Semiring` to `Ring`, but I agree the former is fake generality, so I'll change it